server/planning/search/eventsplanning_search.py

```diff
@@ -296,9 +299,9 @@ def search_by_filter_id(self, filter_id, args=None, page=1, page_size=None, proj
 
         args["filter_id"] = filter_id
 
-        return self.search_repos(search_filter["item_type"], args, page, page_size, projections)
+        return await self.search_repos(search_filter["item_type"], args, page, page_size, projections)
 
-    def get_locked_items(self, repo=None, page=None, page_size=None, projections=None):
+    async def get_locked_items(self, repo=None, page=None, page_size=None, projections=None):
```

Comment:
As this function is only used if client arg `lock_state` is used, we can keep it sync here and comment to upgrade to async when adding support for [search param async callbacks](https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/search/queries/common.py#L487)